final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,130 @@
+import sys
+import requests
+import argparse
+import json
+import logging
+from bs4 import BeautifulSoup
+
+
+log = logging.getLogger("Log")
+
+
+def find_news(url):
+    """Parses news"""
+    logging.info("Searching for news on " + url)
+    try:
+        resp = requests.get(url)
+        soup = BeautifulSoup(resp.content, features="xml")
+        items = soup.findAll('item')
+        log.info(str(len(items)) + " pieces of news have been found")
+    except requests.exceptions.MissingSchema:
+        log.error("Wrong URL")
+        print("Wrong URL: " + url)
+        sys.exit()
```

Comment:
Here and below: It is better to have only one exit point from the application. For example you can raise your custom exception here and handle it in the main function